properties.ui

```diff
@@ -11,11 +11,15 @@
    </rect>
   </property>
   <property name="windowTitle">
-   <string>Dialog</string>
+   <string>Properties Dialog</string>
```

Comment:
Do we want the `Dialog` part at all? For me just `Properties` sounds better. (I assume this is the window title visible to the user, not some internal name.

Reply:
This is the default title of the dialog.
In normal circumstances it should be overwritten by the ID (type of) whatever is shown in the dialog at that moment.

Yes, "Properties" is sufficient. We do not need to mention it is a dialog.

I plan to rework that dialog completely next year. There is some unused LineEdit on top, probably introduced to search in the properties. The plan is to try that out.